src/foremast/securitygroup/create_securitygroup.py

```diff
@@ -71,8 +71,8 @@ def __init__(self, app=None, env=None, region=None, prop_path=None):
         self.region = region
 
         self.properties = get_properties(properties_file=prop_path, env=self.env, region=self.region)
-        generated = get_details(app=self.app_name)
-        self.group = generated.data['project']
+        self.generated = get_details(app=self.app_name)
```

Comment:
Not sure if there will ever be a case where `self.generated` != `generated`. Can't think of anything of the top of my head, so might be a non-issue, but @e4r7hbug thoughts?

Reply:
I like exposing the Instance Attribute in case someone wants to override.